### ISSUE TITLE
Multiple custom attributes of the same type 'Authorize Policy' found.

### ISSUE THREAD

**Issue by Motirck** [Dorin-Mocan/SignalRSwaggerGen#48]
**[Exception]:** System.Reflection.AmbiguousMatchException: Multiple custom attributes of the same type 'Authorize Policy: Policy 1' found.

**Hub method:**

```
[Authorize("Policy 1")]
[Authorize("Policy 2")]
public async Task GetBooks(BookDto dto)
{
    ...
}
```

**What happened?** An exception was thrown because the hub has methods with more than one [Authorize] attribute as it needs more than one policy. [Attribute] only supports one policy at a time.

**Desired behavior:** SignalRSwaggerGen handles more than one [Authorize] attribute for the Hub and its methods

Thank you!

**Comment by Dorin-Mocan** [comment]
@Motirck hi! Working on it.

**Comment by Dorin-Mocan** [comment]
@Motirck I released the version 4.7.0 which should fix your problem. Please check out. And please toss a star if you find this package helpful. Thanks for contribution.

**Comment by Motirck** [comment]
Thank you very much @Dorin-Mocan! It works perfectly now 😄
